src/LightPanels/LightPanels.js

```diff
@@ -183,17 +183,28 @@ module.exports = kind(
 	],
 
 	/**
-	* @method
 	* @private
 	*/
-	create: kind.inherit(function (sup) {
-		return function () {
-			sup.apply(this, arguments);
-			this.updateTransforms();
-			this.orientationChanged();
-			this.directionChanged();
-		};
-	}),
+	create: function () {
+		Control.prototype.create.apply(this, arguments);
+		this.updateTransforms();
+		this.orientationChanged();
+		this.directionChanged();
+		this.setupTransitions();
+	},
+
+	initComponents: function () {
+		Control.prototype.initComponents.apply(this, arguments);
+		if (!this.components || !this.components.length) this.index = -1;
```

Comment:
The assumption here that LightPanel instances would be declared via components isn't necessarily valid. If you create a subkind of LightPanels which included it's own internal panels, those wouldn't be declared in `components`. Suggest we move this to `create()` (so there's a logical opportunity to create panels by a subkind) and use `this.getPanels()` instead of the design-time `components` property.